core/client/fs/src/main/java/alluxio/client/file/cache/CacheManager.java

```diff
@@ -84,17 +84,16 @@ public int getValue() {
     public static CacheManager get(AlluxioConfiguration conf) throws IOException {
       // TODO(feng): support multiple cache managers
       if (CACHE_MANAGER.get() == null) {
-        if (CACHE_INIT_LOCK.tryLock()) {
-          try {
-            if (CACHE_MANAGER.get() == null) {
-              CACHE_MANAGER.set(create(conf));
-            }
-          } catch (IOException e) {
-            Metrics.CREATE_ERRORS.inc();
-            throw new IOException("Failed to create CacheManager", e);
-          } finally {
-            CACHE_INIT_LOCK.unlock();
+        CACHE_INIT_LOCK.lock();
+        try {
+          if (CACHE_MANAGER.get() == null) {
+            CACHE_MANAGER.set(create(conf));
           }
+        } catch (IOException e) {
+          Metrics.CREATE_ERRORS.inc();
+          throw new IOException("Failed to create CacheManager", e);
+        } finally {
+          CACHE_INIT_LOCK.unlock();
         }
```

Comment:
With `LockResource` we don't need `finally`
```suggestion
        }
```

Reply:
Done